resolwe/flow/models/data.py

```diff
@@ -196,7 +198,11 @@ def save_dependencies(self, instance, schema):
         def add_dependency(value):
             """Add parent Data dependency."""
             try:
-                self.parents.add(Data.objects.get(pk=value))  # pylint: disable=no-member
+                DataDependency.objects.update_or_create(
+                    parent=Data.objects.get(pk=value),
+                    child=self,
+                    defaults={'kind': DataDependency.KIND_IO},
```

Comment:
Is there a reason why you don't set a default in `DataDependency.kind` field?

Reply:
Yes, to force the caller to explicitly set a kind and not assume anything by default.